src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/DateTimePicker/DateTimePicker.jsx

```diff
@@ -11,27 +12,47 @@ const variantMapper = {
 };
 
 export const DateTimePicker = ({id, field, editorContext}) => {
+    const displayDateFormat = field.formDefinition.selectorOptions.find(option => option.name === 'format');
+    const variant = variantMapper[field.formDefinition.selectorType];
+    const disabledDays = [];
+    if (field.formDefinition.valueConstraints && field.formDefinition.valueConstraints.length > 0) {
+        const constraints = extractRangeConstraints(field.formDefinition.valueConstraints[0].value.string);
+        let lowerBoundary = constraints.lowerBoundary;
+        let boundaries = {};
+        if (lowerBoundary && lowerBoundary.length > 0) {
+            boundaries.before = {date: new Date(lowerBoundary), include: !constraints.includeLowerBoundary};
```

Comment:
it was making sense because the source objects were the oposite from each other, but I can understand the confusion, `!constraints.includeLowerBoundary`  has been renamed to `constraints.disabledLowerBoundary`